testapps/NuGet.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="LocalArtifacts" value="../artifacts/build" />
+    <add key="LocalArtifacts" value="../artifacts" />
```

Comment:
We require the artifacts/build structure for Universe to be able to pick up packages.